types/meteor-synced-cron/index.d.ts

```diff
@@ -0,0 +1,85 @@
+// Type definitions for littledata:synced-cron 1.5
```

Comment:
```suggestion
// Type definitions for non-npm package littledata:synced-cron 1.5
```

Reply:
Hi @peterblazejewicz,

Thank you so much for the review and change suggestion!

Looks like I did not understand the dts lint error properly... 😕 

Changing to this raises another error, so I will accept the change and fix it after.

Thanks again!